scripts/forking/template/whitelabel/__init__.py

```diff
@@ -0,0 +1,31 @@
+from typing import (
+    Type,
+)
+
+from eth.rlp.blocks import BaseBlock
+from eth.vm.forks.constantinople import (
+    ConstantinopleVM,
+)
+from eth.vm.state import BaseState
+
+from .blocks import IstanbulBlock
+from .headers import (
+    compute_istanbul_difficulty,
+    configure_istanbul_header,
+    create_istanbul_header_from_parent,
+)
+from .state import IstanbulState
+
+
+class IstanbulVM(ConstantinopleVM):
```

Comment:
I'm thinking these files should be templated rather than "white labeled"

```python
from eth.vm.forks.<% parent_fork_snake_case %> import (
    <% parent_fork_name %>VM,
)

class <% fork_name %>VM(<% parent_fork_name %>VM):
    ...
```

This makes it future proof, otherwise we will likely run into a scenario where the find-replace regexing replaces text somewhere like a comment or something....

Reply:
>I'm thinking these files should be templated rather than "white labeled"

Yeah, I also thought about that. The one downside is that you'd lose the ability to run simple flake8/mypy checks against it. You'd have to actually create a fork and run flake8/mypy checks against the result. Not necessarily bad though.

Anyway, this PR is just a very early draft anyway. But thanks for reviewing anyway (also to @veox :+1: )